db/migrate/20190218144231_add_helpful_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddHelpfulToComments < ActiveRecord::Migration[5.2]
+  def change
+    add_column :comments, :helpful, :integer, :default => Comment::HELPFUL['Kind of']
```

Comment:
I'd suggest removing the default value. That way we'll be able to see which comments haven't had feedback.